app/messages/Messages.scala

```diff
@@ -1769,9 +1764,59 @@ object Messages {
       }
     }
 
+    val `Call the debt management helpline`: Message = Message(
+      english = "Call the debt management helpline",
+      welsh   = "Ffoniwch y llinell gymorth rheoli dyledion"
+    )
+
+    val `Telephone...`: Message = Message(
+      english = "Telephone: <strong>0300 123 1813</strong>",
+      welsh   = "Ffôn: <strong>0300 200 1900</strong>"
+    )
+
+    val `Outside UK...`: Message = Message(
+      english = "Outside UK: <strong>+44 2890 538 192</strong>",
+      welsh   = "O’r tu allan i’r DU: +44 300 123 1813"
+    )
+
+    val `Our phone line opening hours...`: Message = Message(
+      english = "Our phone line opening hours are:",
+      welsh   = "Oriau agor ein llinell ffôn yw:"
+    )
+
+    val `Monday to Friday...`: Message = Message(
+      english = "Monday to Friday: 8am to 6pm",
+      welsh   = "Dydd Llun i ddydd Gwener: 8:30 i 17:00"
+    )
+
+    val `Closed weekends...`: Message = Message(
+      english = "Closed weekends and bank holidays.",
+      welsh   = "Ar gau ar benwythnosau a gwyliau banc."
+    )
+
+    val `Text service`: Message = Message(
+      english = "Text service",
+      welsh   = "Gwasanaeth Text Relay"
+    )
+
+    def `Use Relay UK...`(link: String): Message = Message(
+      english = s"""Use Relay UK if you cannot hear or speak on the telephone, dial <strong>18001</strong> then <strong>0345 300 3900</strong>. Find out more on the <a href="$link" class="govuk-link">Relay UK website (opens in new tab)</a>.""",
```

Comment:
I think we need `rel="noreferrer noopener"` for links that open in a new tab (see [here](https://github.com/hmrc/accessibility/blob/main/docs/common-issues/open-in-new-window-links.md))